events/models.py

```diff
@@ -1419,6 +1434,41 @@ class Meta:
         managed = False
 
 
+class XEventFullText(models.Model):
```

Comment:
Even though we're still experimenting, I'd leave the X out of the name and call this `EventFullText` - we can use the x in the api still though 👍

Reply:
`EventFullText` is already reserved/used for the representation of the materialized view used in current full-text search. So the idea was not to touch that right now, as it will be major refactoring and also we would like to use the materialized view for reference testing purposes for now.

Reply:
Agreed, that we'll use a completely new model name here: `EventSearvhIndex`.